runelite-client/src/main/java/net/runelite/client/plugins/achievementdiary/DiaryRequirementsPlugin.java

```diff
@@ -242,7 +244,25 @@ private GenericDiaryRequirement getRequirementsForTitle(String title)
 			{
 				RequirementStringBuilder requirementStringBuilder = new RequirementStringBuilder(i);
 
-				int realSkillLevel = client.getRealSkillLevel(i.getSkill());
+				Skill skill = i.getSkill();
+				int realSkillLevel;
+				//if skill is null assumed to be combat.
+				if (skill == null)
+				{
+					realSkillLevel =  Experience.getCombatLevel(
```

Comment:
`realSkillLevel = client.getLocalPlayer().getCombatLevel();`

Reply:
lol, i knew there had to be an easier way to grab the players combat level.